buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -122,11 +122,11 @@ private object Libraries {
     // https://developer.android.com/kotlin/ktx
 
     // https://developer.android.com/jetpack/androidx/releases/core
-    const val CORE_KTX = "androidx.core:core-ktx:1.6.0"
+    const val CORE_KTX = "androidx.core:core-ktx:1.7.0"
 
     // Lifecycle
     // https://developer.android.com/jetpack/androidx/releases/lifecycle
-    private const val LIFECYCLE_VERSION = "2.3.1"
+    private const val LIFECYCLE_VERSION = "2.4.0"
```

Comment:
@UlisesXumak @IvanTelisman Fyi, updating this dependency required updating the compile/target SDK to api 31 (Android 12). This was done to prep for future `repeatOnLifecycle` Flow api usage (see https://developer.android.com/topic/libraries/architecture/coroutines#restart)